packages/ui/src/components/Button/style.less

```diff
@@ -322,3 +323,111 @@ a.condo-btn {
     @condo-button-border-width);
   }
 }
+
+.condo-btn-dropdown {
+  cursor: default;
+
+  &.condo-btn > span {
+    margin-right: auto;
+  }
+
+  .condo-btn-vertical-divider {
+    z-index: 1;
+    margin-right: 16px;
+    margin-left: 20px;
+    opacity: 0.1;
+  }
+
+  &.condo-btn-block {
+    .condo-btn-vertical-divider {
+      margin-left: auto;
+    }
+  }
+
+  &.condo-btn-secondary.condo-btn-dangerous {
+    .condo-btn-vertical-divider {
+      color: @condo-global-color-red-5;
+    }
+
+    &:hover {
+      .condo-btn-vertical-divider {
+        color: @condo-global-color-red-7;
+      }
+    }
+  }
+
+  .condo-btn-ellipsis {
+    z-index: 1;
+    display: inline-flex;
+  }
+}
+
+.condo-btn-dropdown-wrapper {
+  .condo-btn-dropdown-menu {
+    max-width: 300px;
+    max-height: min(100vh, 360px);
+    padding: 0;
+    overflow: auto;
+    border-radius: @condo-button-border-radius;
+    box-shadow: @condo-global-box-shadow-default;
+
+    .condo-dropdown-menu-item-divider {
+      margin: 0;
+    }
+
+    .condo-dropdown-menu-item {
+      padding: 20px;
+
+      &:hover {
+        background-color: @condo-global-color-white;
+      }
+
+      .condo-dropdown-item-icon {
+        position: relative;
+        top: 1px;
+        display: inline-flex;
+      }
+
+      .condo-typography {
+        /* stylelint-disable-next-line value-no-vendor-prefix */
+        display: -webkit-box;
+        overflow: hidden;
+        -webkit-line-clamp: 2;
```

Comment:
Shouldn't it been added automatically by autoprefixer?

Reply:
It doesn't work properly automatically